README.md

```diff
@@ -83,13 +83,13 @@ can be found [here](https://gcc.gnu.org/onlinedocs/gfortran/OpenMP.html).
 
 * **Download**:
   ```
-  wget "https://www.dropbox.com/s/74ptrnk4k5qcc6o/pandora-linux-precompiled_v0.8.0_beta?dl=1" -O pandora-linux-precompiled_v0.8.0_beta
+  wget "https://www.dropbox.com/s/ltq2gti9t6wav1j/pandora-linux-precompiled_v0.8.1_beta?dl=1" -O pandora-linux-precompiled_v0.8.1_beta
   ```
   * **TODO: updated to a github link when we make the release;**
 * **Running**:
 ```
-chmod +x pandora-linux-precompiled_v0.8.0_beta
-./pandora-linux-precompiled_v0.8.0_beta -h
+chmod +x pandora-linux-precompiled_v0.8.1_beta
+./pandora-linux-precompiled_v0.8.1_beta -h
```

Comment:
This is unnecessary as we haven't even released v0.8.0 yet

Reply:
That is true, v0.8.0 is the one that has the build process changes and provide the static binary, v0.8.1 is the one with asserts refactored. Will fix this.

Reply:
Well, given none of those things have been officially released it is all v0.8.0...

Reply:
Both are fine for me, was my mistake pushing my private versioning. I guess will be able to merge this next week, will have the fix

Reply:
We are now at version [0.8.0-alpha](https://github.com/rmcolq/pandora/releases/tag/v0.8.0-alpha)